### PR TITLE
Update Gradle to 7.0, Bump CI stuff

### PR DESCRIPTION
**Pull request by actuallyasmartname**
(ignore the Merge Pull Request Commit)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -25,8 +25,9 @@ jobs:
       - name: validate gradle wrapper
         uses: gradle/wrapper-validation-action@v1
       - name: setup jdk ${{ matrix.java }}
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
         with:
+          distribution: zulu
```
Comment (modmuss50) [inline .github/workflows/build.yml:30]: I think I would prefer adopt here

File: gradle.properties [hunk 2]
```diff
@@ -2,7 +2,7 @@
 org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
-	# check these on https://fabricmc.net/use
+	# check these on https://modmuss50.me/fabric.html
```
Comment (modmuss50) [inline gradle.properties:5]: No, I would leave this be until a propper solution is sorted.

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0-bin.zip
```
Comment (modmuss50) [inline gradle/wrapper/gradle-wrapper.properties:3]: This also needs loom 0.7

**Review by modmuss50 — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (actuallyasmartname) [reply .github/workflows/build.yml:30]: Alright.

**Review by modmuss50 — APPROVED** [review]
Changes look good, I will hold onto this for a little while as 0.7 loom is still in beta.

Thanks 👍

**Comment by actuallyasmartname** [comment]
Since https://modmuss50.me/fabric.html now redirects to https://fabricmc.net/versions.html, is it ok to change it to the fabric counterpart?

**Comment by sfPlayer1** [comment]
Yes please, it's still only a makeshift thing but a bit more integrated

File: gradle.properties [hunk 4]
```diff
@@ -2,7 +2,7 @@
 org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
-	# check these on https://fabricmc.net/use
+	# check these on https://fabricmc.net/versions.html
```
Comment (actuallyasmartname) [inline gradle.properties:5]: I hope this is ok..

Comment (sfPlayer1) [reply gradle.properties:5]: Yes the use page is pretty useless for this atm

**Review by sfPlayer1 — APPROVED** [review]
LGTM, thanks!

**Review by modmuss50 — APPROVED** [review]
0.7 is as stable as its going to get, so this is great 👍

**Comment by NoComment1105** [comment]
> 0.7 is as stable as its going to get, so this is great 👍 

Gradle 7.0.1 was release a few hours ago, i believe many bugs were fixed, should this change to gradle 7.0.1 too?

**Comment by modmuss50** [comment]
Yeah, might as well 👍

**Comment by actuallyasmartname** [comment]
Gonna update to 7.0.1 in a bit

**Comment by modmuss50** [comment]
Heads up that 7.0.2 just came out :)

**Comment by actuallyasmartname** [comment]
Just realized it when I saw it on the Gradle releases page, but thank you!

**Comment by actuallyasmartname** [comment]
Should be ready to merge

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**